.github/workflows/do-release.yml

```diff
@@ -0,0 +1,158 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create draft release
+
+jobs:
+  create_release:
+    name: Create the Github Release
+    runs-on: ubuntu-latest
+    needs: [release-osx, release-ubuntu-16-04]
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: false
+      - uses: actions/download-artifact@v1
+        with:
+          name: sgrep-osx-${{ github.sha }}
```

Comment:
If this is a release probably better to have the release semver in the filename instead of the shahash?

Reply:
could have a short hash for develop commits where it's changing but generally agree for semver releases

Reply:
these names don't get exposed. It's just the way we shuffle assets around internally. On the release, they get named properly.